integrator/bosh-release/jobs/wso2ei/templates/conf/datasources/master-datasources.xml

```diff
@@ -0,0 +1,68 @@
+<datasources-configuration xmlns:svns="http://org.wso2.securevault/configuration">
+  
+    <providers>
+        <provider>org.wso2.carbon.ndatasource.rdbms.RDBMSDataSourceReader</provider>
+    </providers>
+  
+    <datasources>
+      
+        <datasource>
+            <name>WSO2_CARBON_DB</name>
```

Comment:
We need to have separate configurations for the registry, user-mgt etc. Shall we introduce those changes as well?

Reply:
Created an issue for this in https://github.com/wso2/pivotal-cf-ei/issues/2. Will fix with a pull request in future.

Reply:
+1